### ISSUE TITLE
Issue with peggysPastels theme

### ISSUE THREAD

**Issue by traverse** [arturbien/React95#231]
So I was trying out React95 today, first off kudos it's awesome 👍 but I ran into an issue with one of the themes. I was trying to make the theme dynamically select-able but ran into the following issue.

```
./node_modules/react95/dist/themes/peggysPastels.js
Module not found: Can't resolve '../utils' in 'path\to\project\node_modules\react95\dist\themes'
```
It seems all the `utils` are missing from the built package when looking through the files. My best guess that it might be something with the rollup config but I haven't used rollup before so not 100% sure.

**Comment by arturbien** [comment]
Hey @traverse! ❤️ can you reproduce the issue on codesandbox?

**Comment by traverse** [comment]
Yep that's no problem! Here you go https://codesandbox.io/s/practical-mountain-n2keq?file=/src/App.js

**Comment by arturbien** [comment]
@traverse yes indeed! both `canvasText` and `checkmark` have wrong rgb colors. Wanna create PR fixing this? :)

**Comment by WesSouza** [comment]
We have fixed this on #289 and it is part of [v3.12.2](https://github.com/arturbien/React95/releases/tag/v3.12.2).
